static/app/components/replays/playerDOMAlert.tsx

```diff
@@ -46,15 +46,19 @@ const DOMAlertContainer = styled('div')`
 
 const DOMAlert = styled('div')`
   display: inline-flex;
-  align-items: center;
+  align-items: flex-start;
   justify-items: center;
   padding: ${space(1)} ${space(2)};
   margin: 0 ${space(1)};
   color: ${p => p.theme.white};
   background-color: ${p => p.theme.blue400};
   border-radius: ${p => p.theme.borderRadius};
   gap: 0 ${space(1)};
-  line-height: 0;
+  line-height: 1em;
+
+  & > svg {
```

Comment:
Can we style the components directly? hard to tell what components this is targetting (IconInfo vs IconClose?)